<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WBSDButtonCounters</class>
 <widget class="QMainWindow" name="WBSDButtonCounters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>959</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
     <property name="leftMargin">
      <number>30</number>
     </property>
     <property name="topMargin">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>5</number>
     </property>
     <property name="spacing">
      <number>2</number>
     </property>
     <item row="26" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>Counter Large mild since start</string>
       </property>
      </widget>
     </item>
     <item row="32" column="0">
      <widget class="QLabel" name="label_27">
       <property name="text">
        <string>Volume counter FFLW strong since start</string>
       </property>
      </widget>
     </item>
     <item row="18" column="2">
      <widget class="QLabel" name="smallMildNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="1">
      <widget class="Line" name="line_29">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="Line" name="line_12">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="Line" name="line_19">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="0">
      <widget class="Line" name="line_28">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="Line" name="line_11">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="12" column="2">
      <widget class="Line" name="line_30">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="Line" name="line_20">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="8" column="2">
      <widget class="Line" name="line_21">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="19" column="2">
      <widget class="QLabel" name="smallNormalNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="22" column="2">
      <widget class="QLabel" name="mediumMildNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Counter Large normal since reset</string>
       </property>
      </widget>
     </item>
     <item row="16" column="2">
      <widget class="QLabel" name="rtdVol">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Counter Medium strong since reset</string>
       </property>
      </widget>
     </item>
     <item row="31" column="0">
      <widget class="QLabel" name="label_28">
       <property name="text">
        <string>Volume counter FFLW normal since start</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="smallStrong">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="20" column="2">
      <widget class="QLabel" name="smallStrongNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Counter Medium mild since reset</string>
       </property>
      </widget>
     </item>
     <item row="7" column="2">
      <widget class="QLabel" name="mediumStrong">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="20" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Counter Small strong since start</string>
       </property>
      </widget>
     </item>
     <item row="24" column="2">
      <widget class="QLabel" name="mediumStrongNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="9" column="2">
      <widget class="QLabel" name="largeMild">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="10" column="2">
      <widget class="QLabel" name="largeNormal">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="28" column="0">
      <widget class="QLabel" name="label_22">
       <property name="text">
        <string>Counter Large strong since start</string>
       </property>
      </widget>
     </item>
     <item row="16" column="0">
      <widget class="QLabel" name="label_16">
       <property name="text">
        <string>Volume counter RTD total since reset</string>
       </property>
      </widget>
     </item>
     <item row="13" column="2">
      <widget class="QLabel" name="fflwMild">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="33" column="0">
      <widget class="QLabel" name="label_25">
       <property name="text">
        <string>Volume counter RTD total since start</string>
       </property>
      </widget>
     </item>
     <item row="26" column="2">
      <widget class="QLabel" name="largeMildNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLabel" name="mediumMild">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="27" column="2">
      <widget class="QLabel" name="largeNormalNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="23" column="2">
      <widget class="QLabel" name="mediumNormalNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="30" column="0">
      <widget class="QLabel" name="label_26">
       <property name="text">
        <string>Volume counter FFLW mild since start</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Counter Medium normal since reset</string>
       </property>
      </widget>
     </item>
     <item row="24" column="0">
      <widget class="QLabel" name="label_19">
       <property name="text">
        <string>Counter Medium strong since start</string>
       </property>
      </widget>
     </item>
     <item row="23" column="0">
      <widget class="QLabel" name="label_18">
       <property name="text">
        <string>Counter Medium normal since start</string>
       </property>
      </widget>
     </item>
     <item row="27" column="0">
      <widget class="QLabel" name="label_21">
       <property name="text">
        <string>Counter Large normal since start</string>
       </property>
      </widget>
     </item>
     <item row="14" column="0">
      <widget class="QLabel" name="label_15">
       <property name="text">
        <string>Volume counter FFLW normal since reset</string>
       </property>
      </widget>
     </item>
     <item row="15" column="0">
      <widget class="QLabel" name="label_14">
       <property name="text">
        <string>Volume counter FFLW strong since reset</string>
       </property>
      </widget>
     </item>
     <item row="13" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Volume counter FFLW mild since reset</string>
       </property>
      </widget>
     </item>
     <item row="22" column="0">
      <widget class="QLabel" name="label_17">
       <property name="text">
        <string>Counter Medium mild since start</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Counter Small normal since reset</string>
       </property>
      </widget>
     </item>
     <item row="28" column="2">
      <widget class="QLabel" name="largeStrongNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="19" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Counter Small normal since start</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="smallNormal">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Counter Large mild since reset</string>
       </property>
      </widget>
     </item>
     <item row="11" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Counter Large strong since reset</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Counter Small strong since reset</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="smallMild">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Counter Small mild since reset</string>
       </property>
      </widget>
     </item>
     <item row="11" column="2">
      <widget class="QLabel" name="largeStrong">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="18" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Counter Small mild since start</string>
       </property>
      </widget>
     </item>
     <item row="30" column="2">
      <widget class="QLabel" name="fflwMildNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="31" column="2">
      <widget class="QLabel" name="fflwNormalNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="32" column="2">
      <widget class="QLabel" name="fflwStrongNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="33" column="2">
      <widget class="QLabel" name="rtdVolNr">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="13" column="1">
      <widget class="QLabel" name="lblUnitLiters0">
       <property name="text">
        <string>liters</string>
       </property>
      </widget>
     </item>
     <item row="14" column="1">
      <widget class="QLabel" name="lblUnitLiters1">
       <property name="text">
        <string>liters</string>
       </property>
      </widget>
     </item>
     <item row="15" column="1">
      <widget class="QLabel" name="lblUnitLiters3">
       <property name="text">
        <string>liters</string>
       </property>
      </widget>
     </item>
     <item row="16" column="1">
      <widget class="QLabel" name="lblUnitLiters4">
       <property name="text">
        <string>liters</string>
       </property>
      </widget>
     </item>
     <item row="30" column="1">
      <widget class="QLabel" name="lblUnitLiters5">
       <property name="text">
        <string>liters</string>
       </property>
      </widget>
     </item>
     <item row="31" column="1">
      <widget class="QLabel" name="lblUnitLiters6">
       <property name="text">
        <string>liters</string>
       </property>
      </widget>
     </item>
     <item row="32" column="1">
      <widget class="QLabel" name="lblUnitLiters7">
       <property name="text">
        <string>liters</string>
       </property>
      </widget>
     </item>
     <item row="33" column="1">
      <widget class="QLabel" name="lblUnitLiters8">
       <property name="text">
        <string>liters</string>
       </property>
      </widget>
     </item>
     <item row="17" column="2">
      <widget class="Line" name="line_42">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="17" column="1">
      <widget class="Line" name="line_41">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="Line" name="line_9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="Line" name="line_10">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="Line" name="line_8">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="17" column="0">
      <widget class="Line" name="line_40">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="15" column="2">
      <widget class="QLabel" name="fflwStrong">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="14" column="2">
      <widget class="QLabel" name="fflwNormal">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QLabel" name="mediumNormal">
       <property name="text">
        <string>-</string>
       </property>
      </widget>
     </item>
     <item row="21" column="0">
      <widget class="Line" name="line_49">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="21" column="1">
      <widget class="Line" name="line_50">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="21" column="2">
      <widget class="Line" name="line_51">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="25" column="2">
      <widget class="Line" name="line_60">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="25" column="1">
      <widget class="Line" name="line_59">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="25" column="0">
      <widget class="Line" name="line_58">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="29" column="0">
      <widget class="Line" name="line_67">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="29" column="1">
      <widget class="Line" name="line_68">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="29" column="2">
      <widget class="Line" name="line_69">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="34" column="0">
      <widget class="Line" name="line_79">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="34" column="1">
      <widget class="Line" name="line_80">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="34" column="2">
      <widget class="Line" name="line_81">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_23">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_24">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label_29">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_30">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_31">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLabel" name="label_32">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QLabel" name="label_33">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLabel" name="label_34">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="11" column="1">
      <widget class="QLabel" name="label_35">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="18" column="1">
      <widget class="QLabel" name="label_36">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="19" column="1">
      <widget class="QLabel" name="label_37">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="20" column="1">
      <widget class="QLabel" name="label_38">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="22" column="1">
      <widget class="QLabel" name="label_39">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="23" column="1">
      <widget class="QLabel" name="label_40">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="24" column="1">
      <widget class="QLabel" name="label_41">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="26" column="1">
      <widget class="QLabel" name="label_42">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="27" column="1">
      <widget class="QLabel" name="label_43">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
     <item row="28" column="1">
      <widget class="QLabel" name="label_44">
       <property name="text">
        <string>cups</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
